testsuite/models/testrun.py

```diff
@@ -0,0 +1,80 @@
+''' Test run module.
+'''
+import json
+
+from testsuite.extensions import db
+
+
+class TestRun(db.Model):
+    ''' The results of a single test run.
```

Comment:
Also now that we've got a  db here it'd be great to capture date, vendor, and some kind of "is a complete test run including all tags" boolean at this stage. 

FEATURE CREEP!

(But let's keep those in mind so we can facilitate reporting, which is currently a manual process.)